src/js/components/VariableEditor.js

```diff
@@ -33,13 +33,85 @@ class VariableEditor extends React.Component {
         parsedInput: {
             type: false,
             value: null
-        }
+        },
+        copied: false
     }
 
     constructor(props) {
         super(props)
     }
 
+    handleCopy = () => {
```

Comment:
this code was almost exactly copied from [./VariableMeta.js](https://github.com/mac-s-g/react-json-view/blob/master/src/js/components/VariableMeta.js#L31)

the copy function is working, but I want to abstract the code into a single location and add a few tests before merging.